en/advanced/best-practice-security.md

```diff
@@ -152,6 +153,16 @@ app.use(session({
 }))
 ```
 
+## Prevent brute-force attacks against authorisation
```

Comment:
"authorization"
Spelling it with an "s" is the British spelling, and by convention we use American spelling